build.mjs

```diff
@@ -0,0 +1,7 @@
+import { writeFile } from 'fs/promises'
+import labels from './labels.mjs'
+
+const configPath = new URL('labels.json', import.meta.url)
```

Comment:
Why is this preferable to `path.join(__dirname, 'labels.json')`?

Reply:
Sadly got removed in ESM


### No `__filename` or `__dirname`
https://nodejs.org/api/esm.html#no-__filename-or-__dirname

>`__filename` and `__dirname` use cases can be replicated via [`import.meta.url`](https://nodejs.org/api/esm.html#importmetaurl).

Reply:
Ah, I didn't know that, thanks.